# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini
# Commit: 920da79f43fc
# Full Hash: 920da79f43fcf5802ebd5eb2bf610a18e821dacc
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-03-19 03:53:34
# Regressor Bug: 1620197
# File Overlap Count: 1
# Description:
#   Bug 1620197 - Enable multiple results from WebAssembly functions r=lth
#   
#   This patch enables multi-value calls and returns, adding some tests, and
#   conditionally disabling a couple expect-fail reftests that now pass.
#   
# ==============================================================================

diff -r 98d7f9e4839f -r 920da79f43fc testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini
--- a/testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini	Wed Mar 18 17:12:14 2020 +0000
+++ b/testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini	Wed Mar 18 16:36:45 2020 +0000
@@ -14,7 +14,8 @@
     expected: FAIL
 
   [multiple return values inside wasm]
-    expected: FAIL
+    expected:
+      if not nightly_build: FAIL
 
   [multiple return values from js to wasm]
     expected: FAIL
